index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+use TheRealMVP\Importer\DBImport;
+use TheRealMVP\Importer\GetAPI;
+use TheRealMVP\Importer\PDO;
+use TheRealMVP\DisplayHelper\DisplayData;
+use TheRealMVP\Hydrators\TeamHydrator;
+
+require_once './vendor/autoload.php';
+
+$pdoConnection = PDO::createPDO();
+$db = new DBImport($pdoConnection, new GetAPI());
+$db->dropTablesAndCreateTables();
+$db->storeData();
```

Comment:
This still results in the database being refreshed on every page load.

Could you remove these three lines, create a new file in the root of the project called importer.php (or something), and move them into that?

That way if you ever need to refresh the database you just go to the importer page and it runs the functionality. No need to display anything at all on the page.